### PR TITLE
UIINREACH-17: INN-Reach General Settings: Central Server Connection C…

### PR DESCRIPTION
**Pull request by Dmytro-Melnyshyn**
[UIINREACH-17](https://issues.folio.org/browse/UIINREACH-17)

### CODE REVIEW

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationForm/CentralServersConfigurationForm.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
```
Comment (Godlevskyi) [inline src/settings/components/CentralServersConfiguration/CentralServersConfigurationForm/CentralServersConfigurationForm.js:1]: Please format on a separate line

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationForm/CentralServersConfigurationForm.js [hunk 2]
```diff
@@ -179,7 +197,7 @@ const CentralConfigurationForm = ({
                     name={CENTRAL_SERVER_CONFIGURATION_FIELDS.NAME}
                     type="text"
                     component={TextField}
-                    label={<FormattedMessage id={`ui-inn-reach.${CENTRAL_SERVER_CONFIGURATION_FIELDS.NAME}`} />}
+                    label={<FormattedMessage id="ui-inn-reach.settings.central-server-configuration.create/edit.field.name" />}
```
Comment (Godlevskyi) [inline src/settings/components/CentralServersConfiguration/CentralServersConfigurationForm/CentralServersConfigurationForm.js:200]: Please do not use "/" in the translation code 
Change it to create-edit

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationForm/components/RepeatableField/FieldRow.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import React, { useRef } from 'react';
+import React, { useEffect, useRef } from 'react';
```
Comment (Godlevskyi) [inline src/settings/components/CentralServersConfiguration/CentralServersConfigurationForm/components/RepeatableField/FieldRow.js:1]: please format on a separater line

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationForm/CentralServersConfigurationForm.js (hunk 1 above)
Comment (Dmytro-Melnyshyn) [reply src/settings/components/CentralServersConfiguration/CentralServersConfigurationForm/CentralServersConfigurationForm.js:1]: Thanks, fixed.

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationForm/CentralServersConfigurationForm.js (hunk 2 above)
Comment (Dmytro-Melnyshyn) [reply src/settings/components/CentralServersConfiguration/CentralServersConfigurationForm/CentralServersConfigurationForm.js:200]: Thanks, fixed.

File: src/settings/components/CentralServersConfiguration/CentralServersConfigurationForm/components/RepeatableField/FieldRow.js (hunk 3 above)
Comment (Dmytro-Melnyshyn) [reply src/settings/components/CentralServersConfiguration/CentralServersConfigurationForm/components/RepeatableField/FieldRow.js:1]: Thanks, fixed.

**Review by Godlevskyi — APPROVED** [review]

**Pull request merged**